app/controllers/api/v1/common_parameters_controller.rb

```diff
@@ -52,6 +53,17 @@ def update
       def destroy
         process_response @common_parameter.destroy
       end
+
+      def resource_class
+        GlobalLookupKey.where(:should_be_global => true)
```

Comment:
doesn't GlobalLookupKey is already Global? what does the boolean mean? e,g, global vs host parameter or global vs smart class params?